samples/error-correction/syndrome/Syndrome.qs

```diff
@@ -2,13 +2,36 @@
 // Licensed under the MIT License.
 
 namespace Microsoft.Quantum.Samples.ErrorCorrection.Syndrome {
+    open Microsoft.Quantum.Arrays;
     open Microsoft.Quantum.Canon;
+    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Diagnostics;
     open Microsoft.Quantum.Intrinsic;
-    open Microsoft.Quantum.Preparation;
     open Microsoft.Quantum.Measurement;
-    open Microsoft.Quantum.Arrays;
-    open Microsoft.Quantum.Diagnostics;
-    open Microsoft.Quantum.Convert;
+    open Microsoft.Quantum.Preparation;
+    open Microsoft.Quantum.Random;
+
+    /// # Summary
+    /// Runs the Syndrome sample.
+    ///
+    /// # Input
+    /// ## qubits
+    /// The number of qubits to use.
+    @EntryPoint()
+    operation RunSyndrome(qubits : Int) : Unit {
+        let qubitIndices = Shuffle(RangeAsIntArray(0 .. qubits - 1));
+        let inputValues = DrawMany(DrawRandomBool, qubits, 0.5);
```

Comment:
Same comment here, this is great but may benefit from some additional comments to help new users understand what the results of these two lines will be.